books/RayTracingInOneWeekend.html

```diff
@@ -1429,8 +1430,56 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     [Listing [rtweekend-initial]: <kbd>[rtweekend.h]</kbd> The rtweekend.h common header]
 
+For all main program files, we will include `rtweekend.h` first, so most other header files (where
```

Comment:
I would rephrase this to:

_All main program files will include `rtweekend.h` first, so most..._

Reply:
Done.